app/controllers/charges_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ChargesController < ApplicationController
+
+  def new
+  end
+
+  def create
+  @amount = 500
+
+  customer = Stripe::Customer.create(
+    :email => 'example@stripe.com',
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.